src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController+Extension.swift

```diff
@@ -122,18 +134,15 @@ extension OnboardingInfoViewController {
 		containerView.addSubview(titleLabel)
 
 		for (index, country) in countries.enumerated() {
-			// MARK: - Create stack view to hold label and flag for country.
 			let stackView = UIStackView()
 			stackView.axis = .horizontal
 			stackView.alignment = .center
 			stackView.spacing = 13
 
-			// MARK: - Country name.
```

Comment:
I agree these marks are too much, maybe we can have a better variable naming here instead, like `countryNameLabel`